### PR TITLE
Scrum 65

### PR DESCRIPTION
**Pull request by Hsbalazs**

### CODE REVIEW

File: frontend/src/pages/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        const path = '/';
+        navigate(path);
+    }
+
+    const handleProfile = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        if (user?.sub == "" || user?.lastName == "" || user?.lastName == "" || password == "") {
+            setErrMessage("Please, fill all field!");
+            return;
+        }
```
Comment (markkovari) [inline frontend/src/pages/Profile.tsx:40]: You can use `required` in the inputs and you don't need to validate if the values are not empty.

File: frontend/src/pages/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    onChange={handleUserChange}
+                />
+                <label htmlFor="password">Password:</label>
+                <input
+                    type="password"
+                    aria-label={"pass"}
+                    name="password"
+                    placeholder={password}
+                    id="password"
+                    defaultValue={password}
```
Comment (markkovari) [inline frontend/src/pages/Profile.tsx:84]: This should be `value`.

```suggestion
             value={password}
```

File: frontend/src/pages/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                    onChange={e => setPassword(e.target.value)}
+                    autoComplete="current-password"
```
Comment (markkovari) [inline frontend/src/pages/Profile.tsx:86]:
```suggestion
                    autoComplete="new-password"
```

and you can use these nice little things for the other fields too.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

File: frontend/src/pages/Profile.tsx (hunk 1 above)
Comment (Hsbalazs) [reply frontend/src/pages/Profile.tsx]: It is nicer solution. :-) Thank you!

File: frontend/src/pages/Profile.tsx (hunk 2 above)
Comment (Hsbalazs) [reply frontend/src/pages/Profile.tsx]: I have done.

File: frontend/src/pages/Profile.tsx (hunk 3 above)
Comment (Hsbalazs) [reply frontend/src/pages/Profile.tsx]: I added autoComplete to all the inputs "email", "given-name", "family-name".

**Review by AnnaUgrai — APPROVED** [review]

**Review by carlos-ponton — APPROVED** [review]

**Pull request merged**